public/scss/components/OrderSummaryBox.module.scss

```
@@ -796,6 +874,152 @@
         }
     }
 
+    &_voucherShowMoreContainer
+    {
+        @include flex(row, center, center);
+        margin: 8px 0 calc(26px + 2.6vh);
+    }
+
+    &_voucherShowMoreButton
+    {
+        @include typographyBuilder(secondary, 400, 12, 16);
+        color: $color_rose;
+        padding: 0;
+        outline: 0;
+        border: 0;
+        cursor: pointer;
+        background: transparent;
+
+        &::after
+        {
+            content: "+";
+            display: inline;
+            margin-left: 12px;
+        }
+    }
+
+    &_voucherDetailPopUpContainer
+    {
+        @include fixed(0, auto, auto, 0);
+        @include flex(row, center, center);
+        @include fixedSize(100%);
+        z-index: 1050;
+        background-color: rgba($color_black, 0.7);
+
+        & > [class*="popupBackgroundClassName"]
+        {
+            display: none;
+            opacity: 0;
+            visibility: hidden;
+        }
+    }
+
+    &_voucherDetailPopUpBody
+    {
+        @include flex(column, flex-start, flex-start);
+        @include fixed(auto, auto, 0);
+        z-index: 1;
+        margin: 0;
+        background-color: $color_white;
+        border-radius: 2px;
+
+        max-width: 100vw;
+        max-height: 80vh;;

```

```suggestion
        max-height: 80vh;

```

titik koma-nya kebanyakan